src/components/TransactionSettingsBottomSheet.tsx

```diff
@@ -69,14 +70,25 @@ const TransactionSettingsBottomSheet: React.FC<
     saveSwapSlippage,
   } = useSwapSettingsStore();
 
+  const { markAsManuallyChanged } = useInitialRecommendedFee(
+    recommendedFee,
+    context,
+  );
+
   const timeoutInfoBottomSheetModalRef = useRef<BottomSheetModal>(null);
   const feeInfoBottomSheetModalRef = useRef<BottomSheetModal>(null);
   const memoInfoBottomSheetModalRef = useRef<BottomSheetModal>(null);
   const slippageInfoBottomSheetModalRef = useRef<BottomSheetModal>(null);
 
   // Derived values based on context
   const memo = context === TransactionContext.Swap ? "" : transactionMemo;
-  const fee = context === TransactionContext.Swap ? swapFee : transactionFee;
+  const storeFee =
+    context === TransactionContext.Swap ? swapFee : transactionFee;
+
+  const fee =
+    storeFee === MIN_TRANSACTION_FEE && recommendedFee
+      ? recommendedFee
+      : storeFee;
```

Comment:
just double checking, do we still need to do this? wouldn't the `storeFee` already be set to the `recommendedFee` through the `useInitialRecommendedFee()` hook?